<!-- -->
<launch>
	
	<arg name="file_path" default="$(find recorder)/data/state/" />
	<arg name="file_name" default="vehicle_state.txt" />

	<node name="record_vehicle_state_node" pkg="recorder" type="record_vehicle_state_node" output= "screen">
		<param name="file_path" value="$(arg file_path)"/>
		<param name="file_name" value="$(arg file_name)"/>
	</node>

</launch>
